cohort-engine-api-web/src/main/java/com/ibm/cohort/engine/api/service/CohortEngineRestHandler.java

```diff
@@ -152,16 +156,17 @@ public CohortEngineRestHandler() {
 		@ApiImplicitParam(name=REQUEST_DATA_PART, value=REQUEST_DATA_PART_DESC, dataTypeClass = MeasureEvaluation.class, required=true, paramType="form"),
 		@ApiImplicitParam(name=MEASURE_PART, value=MEASURE_PART_DESC, dataTypeClass = File.class, required=true, paramType="form", type="file" )
 	})
-	@ApiResponses(value = { 
+	@ApiResponses(value = {
 			@ApiResponse(code = 200, message = "Successful Operation"),
-			@ApiResponse(code = 400, message = "Bad Request", response = ServiceErrorList.class), 
-			@ApiResponse(code = 500, message = "Server Error", response = ServiceErrorList.class) 
+			@ApiResponse(code = 400, message = "Bad Request", response = ServiceErrorList.class),
+			@ApiResponse(code = 500, message = "Server Error", response = ServiceErrorList.class)
 	})
 	public Response evaluateMeasure(@Context HttpServletRequest request,
+
 			@ApiParam(value = ServiceBaseConstants.MINOR_VERSION_DESCRIPTION, required = true, defaultValue = ServiceBuildConstants.DATE) @QueryParam("version") String version,
-			@ApiParam(hidden = true, type="file", required=true) IMultipartBody multipartBody) {		
+			@ApiParam(value = "patients and the measures to run", required = true) IMultipartBody multipartBody) {
```

Comment:
Wanted to double check if this change was intentional since it's in `evaluateMeasure`

Reply:
Good catch, I think this was something Corey and I were messing with yesterday and I messed with weirdly.